docs: Add GraphQL docs for unset field (#853)
ParsePlatform/Docs: _includes/graphql/objects.md

  }
}
```

**Note:** If you use [Apollo Client](https://www.apollographql.com/docs/react/) it's recommended to request the modified fields and `id` during the Mutation, then the [Apollo Client](https://www.apollographql.com/docs/react/) will automatically update its local store and push the new data across your app; i.e. If you update `playerName` you should request `playerName` and `id` like the code above.

### Unset a field

> Across the whole GraphQL API you can simply unset a field by setting his value to `null`.

Reviewer: ```suggestion
Across the whole GraphQL API you can simply unset a field by setting its value to `null`.
```

Following the official GraphQL API Specs, setting a field to `null` through the GraphQL API will completly unset the field in the database on the targeted Parse Object. GraphQL API will transform `null` on the server before saving the object to correctly unset the field into the database.

```js
// Header
{
  "X-Parse-Application-Id": "APPLICATION_ID",
  "X-Parse-Master-Key": "MASTER_KEY" // (optional)
}
```

```graphql
# GraphQL
mutation updateAGameScore {
  updateGameScore(
    input: { id: "R2FtZVNjb3JlOmM3TVpDZEhQY2w=", fields: { playerName: null } }
  ) {
    gameScore {
      id
      playerName
    }
  }
}
```

```js
// Response
{
  "data": {
    "updateGameScore": {
      "gameScore": {
        "id": "R2FtZVNjb3JlOmM3TVpDZEhQY2w=",
        "playerName": null
      }
    }
  }
}
```

> GraphQL API will always return `null` if the field is `null` or `undefined` in the database. A GraphQL API do not support difference between `null` and `undefined` into the data response.

Reviewer: ```suggestion
The GraphQL API will always return `null` if the field is `null` or `undefined` in the database. The GraphQL API does not differentiate between `null` and `undefined` in the data response.
```

```
**Note:** If you use [Apollo Client](https://www.apollographql.com/docs/react/) it's recommended to request the modified fields and `id` during the Mutation, then the [Apollo Client](https://www.apollographql.com/docs/react/) will automatically update its local store and push the new data across your app; i.e. If you update `playerName` you should request `playerName` and `id` like the code above.

### Unset a field
Across the whole GraphQL API you can simply unset a field by setting his value to `null`.

> **Note:** Following the official GraphQL API Specs, setting a field to `null` through the GraphQL API will completly unset the field in the database on the targeted Parse Object. GraphQL API will transform `null` on the server before saving the object to correctly unset the field into the database.

Reviewer: > Following the official GraphQL API specification, setting a field to `null` will unset the field in the database for the  respective Parse Object. The GraphQL API will transform `null` on the server side before saving the object to correctly unset the field in the database.

Remove `Note:` before the comment, doesn't seem to be necessary.

```js
// Header
{
  "X-Parse-Application-Id": "APPLICATION_ID",
  "X-Parse-Master-Key": "MASTER_KEY" // (optional)
}
```
```graphql
# GraphQL
mutation updateAGameScore {
  updateGameScore(
    input: {
      id: "R2FtZVNjb3JlOmM3TVpDZEhQY2w="
      fields: { playerName: null }
    }
  ) {
    gameScore {
      id
      playerName
    }
  }
}
```
```js
// Response
{
  "data": {
    "updateGameScore": {
      "gameScore": {
        "id": "R2FtZVNjb3JlOmM3TVpDZEhQY2w=",
        "playerName": null
      }
    }
  }
}
```

> **Additional note:** GraphQL API will always return `null` if the field is `null` or `undefined` in the database. A GraphQL API do not support difference between `null` and `undefined` into the data response.

Reviewer: > The GraphQL API will return `null` if the field is `null` or `undefined` in the database. The GraphQL API do not differentiate between `null` and `undefined` in the data response.

Remove `Additional note:` before the comment, doesn't seem to be necessary.
